src/ray/common/ray_config_def.h

```diff
@@ -902,3 +902,10 @@ RAY_CONFIG(int, object_manager_client_connection_num, 4)
 //     std::min(std::max(2, num_cpus / 4), 8)
 // Update this to overwrite it.
 RAY_CONFIG(int, object_manager_rpc_threads_num, 0)
+
+// If autoscaler doesn't set draining deadline,
+// the default value is the time when the draining
+// request is received + default_draining_period_s
+// TODO(jjyao): Remove this after autoscaler always sets
+// draining deadline.
+RAY_CONFIG(int64_t, default_draining_period_ms, 60000)
```

Comment:
Doesn't it make more sense we don't have any opinion here? 

Also, I each drain request set the deadline right?